### PR TITLE
Add resource attributes

### PR DESCRIPTION
**Pull request by theletterf**

### CODE REVIEW

**Comment by Rauno56** [comment]
:+1: This becomes reality with https://github.com/signalfx/splunk-otel-js/pull/312
I will leave it open for now.

**Review by Rauno56 — APPROVED** [review]

File: docs/correlate-logs-traces.md [hunk 1]
```diff
@@ -11,11 +11,12 @@ The following logging librares are supported:
...
 - Winston
 
-## Available trace data
+## Injected trace data
 
-The following attributes are available for applications instrumented using the Splunk distribution of OpenTelemetry JS:
+The following attributes are automatically injected into logs:
 
 - Trace information: `trace_id`, `span_id`, and `trace_flags`
+- Resource attributes: `service.name`, `service.version`, and `service.environment`
```
[the remark this answers is not in the document]
Comment (theletterf) [reply docs/correlate-logs-traces.md:19]: So we're good to go?

[the remark this answers is not in the document]
Comment (seemk) [reply docs/correlate-logs-traces.md:19]: Took a quick look, so some clarification: `deployment.environment` resource attribute is injected into the logs as `service.environment`. I think there is no clear OTel convention yet how this should be done. There is https://github.com/open-telemetry/oteps/pull/114 according to which the resource attributes should be prefixed with `resource.` in the logs, but this is just a proposal :shrug: 

| Resource attribute | Log field |
| ------------------------- | ----------- |
| `deployment.environment` | `service.environment` |
| `service.name` | `service.name` |
| `service.version` | `service.version` |

[the remark this answers is not in the document]
Comment (theletterf) [reply docs/correlate-logs-traces.md:19]: @Rauno56 ?

[the remark this answers is not in the document]
Comment (Rauno56) [reply docs/correlate-logs-traces.md:19]: It'd make more sense to have them prefixed by `resource.` to me. I haven't changed the behavior from the initial implementation.

[the remark this answers is not in the document]
Comment (seemk) [reply docs/correlate-logs-traces.md:19]: I propose to leave the log fields as is currently and in this PR only change the wording in docs from `service.environment` to `deployment.environment`.

So the input is documented, we can change the log fields output in a separate PR.

[the remark this answers is not in the document]
Comment (Rauno56) [reply docs/correlate-logs-traces.md:19]: How about:
```suggestion
- Resource attributes: `service.name`, `service.version`, and `deployment.environment`(as `service.environment`)
```

**Pull request merged**